cura/CuraApplication.py

```diff
@@ -925,3 +926,8 @@ def addAdditionalComponent(self, area_id, component):
         self._additional_components[area_id].append(component)
 
         self.additionalComponentsChanged.emit(area_id)
+
+    @pyqtSlot(str)
```

Comment:
This is pretty nasty debug code. If you want to use this more often, i'd suggest moving the import out of the function.